readme.md

```diff
@@ -36,7 +36,7 @@ The functions documented below can be imported by name, and used in your `config
 
 ### addBabelPlugin(plugin)
 
-Adds a babel plugin. Whatever you pass for `plugin` will be added to Babel's `plugins` array. Consult their docs for more info.  
```

Comment:
Trailing spaces are meaningful in markdown

Reply:
Sorry, my editor trims them automatically and I don't find trailing spaces anywhere else. I 'll restore them by another commit.

Reply:
FIY 2 trailing spaces in markdown mean "newline", while an empty line means "new paragraph"

Reply:
Yeah I've never used two trailing spaces (<br />) before.